mbf_abstract_nav/src/abstract_navigation_server.cpp

```diff
@@ -73,11 +73,16 @@ AbstractNavigationServer::AbstractNavigationServer(const TFPtr &tf_listener_ptr)
   oscillation_timeout_ = ros::Duration(oscillation_timeout);
   private_nh_.param("oscillation_distance", oscillation_distance_, 0.02);
 
+  private_nh_.param("odom_topic", odom_topic_, std::string("odom"));
```

Comment:
No param needed. Use the standard `odom` and can remap if different.

Reply:
Because I saw so many open source ros package also provide topic name parameter to let user change, but I think I can follow this package rule.

Reply:
Many packages are doing that, but I think that is an anti-pattern